src/Ginger/Translation.elm

```diff
@@ -80,68 +81,88 @@ type Translation
 
 
 type alias Translations =
-    { en : String
-    , nl : String
-    , de : String
-    , zh : String
-    }
+    Dict String String
+
+
+type Text
```

Comment:
Currently unused, I'm thinking about parsing to html during the decoding phase, this might actually result in a significant runtime perf improvement. Parsing happens on every animation frame in some situations :(

Reply:
I don't see much issues applying this to internal functions in this package because we know which fields in the REST API might contain HTML and which don't.

It does cause some 'issues' to functions like `fromList` in this module, as in we don't know if the list contains HTML or not.

Possible solution:
```elm
fromHtmlList : List (Language, String) -> Maybe Translation

fromHtmlList : List (Language, String) -> Result ParseError Translation

fromList : List (Language, String) -> Translation
fromList xs =
  just try to parse and use Plain on failure
```

Reply:
We could also always use Plain for `fromList` and parse when you call `html`. Which would mean there is no benefit for Translations created outside of this package but the user would notice a thing since the whole thing is opaque. 

html would become something like
```elm

{-| Translate and render as Html markup
-}
html : Language -> Translation -> List (Html msg)
html language translation =
   case toStringEscaped language translation of
      Markup x -> x
      Plain x->  Internal.Html.toHtml x
```